app/assets/stylesheets/application.scss

```diff
@@ -46,3 +46,21 @@ p {
   height: 1px;
   overflow: hidden;
 }
+
+.hide-bullet-points {
+  list-style: none;
+}
+
+.hide-bullet-points li{
+  margin: 5px 0;
+}
+
+.mobile-display-only{
```

Comment:
I'm not sure I explained my thinking last time.

So in my eyes why does "mobile-display-only" mean something should be white in colour?  I believe you should name your class to reflect what you are trying to style.

Reply:
Resolved this in 1dfe7e1